am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GetRolePermissionsIntegrationTest.java

```diff
@@ -42,13 +45,23 @@ void setUp() {
 
     @Test
     void whenGettingRolePermissionsShouldReturnPermissionsAndSecurityClassificationsForSpecifiedRole() {
+
+        ImmutableSet multiPermission = ImmutableSet.builder().add(READ).add(CREATE).add(UPDATE).build();
```

Comment:
```suggestion
        Set<Permission> multiPermission = ImmutableSet.of(READ, CREATE,UPDATE);
```

Reply:
although personally I think this is uneccessary complication for the test. The logic does nothing with permissions, so tbh I would actually remove